### DISCUSSION TITLE
Edubuntu and LTSP?

### DISCUSSION THREAD

**Discussion by JB-Prof** [ltsp/ltsp#814] [General]
Hello!

Many long-time LTSP users may recall that LTSP used to be integrated with Edubuntu (an education-focussed spin of Ubuntu -- I think I started using that version in 2005).  It was extremely useful as LTSP is widespread in educational settings even until today.  Edubuntu was discontinued with the last released in 2014.

I just received an email on the old Edubuntu mailing list (which I forgot was still active) that a new beta has now been released! 

https://cdimages.ubuntu.com/edubuntu/releases/23.04/beta/

Given this is a spin of Ubuntu, would it be safe to say that the current version of LTSP would integrate seamlessly with it?  If not, could it be with minimal modifications?

Thank you.

**Comment by alkisg** [comment]
Sure, the new LTSP should work with any Ubuntu flavor.
